### DISCUSSION TITLE
composer is not installed

### DISCUSSION THREAD

**Discussion by Spu4ka** [wintercms/winter#167] [Q&A] [answered]
after token input
gives an error message

[RuntimeException]
  Failed to clone https://github.com/composer/installers.git, git was not found, check th
  at it is installed and in your PATH env.

  'git' is not recognized as an internal or external command,
  operable program or batch file.`

**Comment by LukeTowers** [comment, marked as answer]
You need to make sure that git is installed on your system and is accessible to composer.
